src/code/z_play.c

```diff
@@ -538,13 +538,13 @@ void Play_AssignPlayerActorCsIdsFromScene(GameState* thisx, s32 startActorCsId)
     }
 }
 
-// These regs are used by Gameplay_Draw, and several actors, purpose as yet unclear.
-void func_8016A268(GameState* thisx, s16 arg1, u8 arg2, u8 arg3, u8 arg4, u8 arg5) {
-    MREG(64) = arg1;
-    MREG(65) = arg2;
-    MREG(66) = arg3;
-    MREG(67) = arg4;
-    MREG(68) = arg5;
+// Set values to fill screen
+void Play_FillScreen(GameState* thisx, s16 fillScreenOn, u8 red, u8 green, u8 blue, u8 alpha) {
```

Comment:
Oh at last! Might as well make this a proper Doxygen comment.

Reply:
I stole the comment from the z_play branch. I say poke the z_play guys to properly document it c:

Reply:
I was the one who tested and documented this function, so you can poke at me for this lazy comment.

Reply:
_pokes_